lib/segment/src/types.rs

```diff
@@ -661,6 +664,47 @@ impl From<BinaryQuantizationConfig> for QuantizationConfig {
     }
 }
 
+#[derive(Debug, Deserialize, Serialize, JsonSchema, Validate, Clone, PartialEq, Default)]
+pub struct StrictModeConfig {
```

Comment:
At this time the -Diff struct of this appears to be exactly the same. In that case, does it make sense to use a diff structure at all?